simapp/v2/export.go

```diff
@@ -1,10 +1,30 @@
 package simapp
 
 import (
-	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"context"
+
+	serverv2 "cosmossdk.io/server/v2"
 )
 
-// ExportAppStateAndValidators exports the state of the application for a genesis file.
-func (app *SimApp[T]) ExportAppStateAndValidators(forZeroHeight bool, jailAllowedAddrs, modulesToExport []string) (servertypes.ExportedApp, error) {
-	panic("not implemented")
+// ExportAppStateAndValidators exports the state of the application for a genesis
+// file.
+func (app *SimApp[T]) ExportAppStateAndValidators(forZeroHeight bool, jailAllowedAddrs, modulesToExport []string) (serverv2.ExportedApp, error) {
```

Comment:
With this change I would have expected some other changes in simapp `commands.go` and x/gentuil